doc/cross-compile/buildroot.md

```diff
@@ -0,0 +1,39 @@
+# How to make cross-compilation work with buildroot
+
+1. install the build dependencies on your host PC, like: cmake, g++, clang...
+2. install the `rust` toolchain:
+
+   ```console
+   curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh
+   ```
+
+3. add the arm64 target for Rust: `rustup target add aarch64-unknown-linux-gnu`
+4. modify the arm64 target name to suit your cross-compilation tool, need to
+   create this file `~/.cargo/config.toml`, add:
+   ```console
+   [target.aarch64-unknown-linux-gnu]
+   linker = "aarch64-buildroot-linux-gnu-gcc"
+   ```
+   "aarch64-buildroot-linux-GNU-gcc" is your real cross-compilation tool name
+5. source your cross-compilation buildroot environment:
+   `source /to/your/environment-setup`, this file should be in your buildroot
+   folder
+
+6. add the buildroot sysroot on host PC environment:
+   ```console
+   export BINDGEN_EXTRA_CLANG_ARGS="--sysroot=/to/your/sysroot"
+   ```
+7. Change to the iceoryx2 directory
+
+   ```console
+   cd iceoryx2
+   ```
+
+8. cmake -S . -B build -DBUILD_EXAMPLES=ON -DCMAKE_INSTALL_PREFIX=../\_OUTPUT
+   -DRUST_TARGET_TRIPLET='aarch64-unknown-linux-GNU'
+
+9. make -j8
+
+10. make install
```

Comment:
````suggestion
8. Configure, build and install iceoryx2
   ```console
   cmake -S . -B build -DBUILD_EXAMPLES=ON -DCMAKE_INSTALL_PREFIX=../\_OUTPUT -DRUST_TARGET_TRIPLET='aarch64-unknown-linux-GNU'
   cmake --build build
   cmake --install build
   ```
````

Does it also work by just invoking cmake? If not, please revert to using `make`

Reply:
About the `make`, because my `cmake` can't accept this command: `cmake --build build`, so I use `make` instead.
```
$ cmake --build build
CMake Error: Unknown argument --build
CMake Error: Run 'cmake --help' for all supported options.

$ cmake --version
cmake version 3.22.1

CMake suite maintained and supported by Kitware (kitware.com/cmake).
```

Anyway, I use your suggestion to use `cmake`. I think your `cmake` is well for `--build`.